Orange/data/variable.py

```diff
@@ -232,51 +32,47 @@ def __new__(cls, name, bases, attrs):
         return obj
 
 
-class Variable(metaclass=VariableMeta):
+class Variable(str, metaclass=VariableMeta):
     """
     The base class for variable descriptors contains the variable's
-    name and some basic properties.
-
-    .. attribute:: name
+    name and some basic properties. This extends str so it plays nicely with
+    pandas' column values.
 
+    Attributes
+    ----------
+    name : str
         The name of the variable.
-
-    .. attribute:: unknown_str
-
+    unknown_str : set
         A set of values that represent unknowns in conversion from textual
-        formats. Default is `{"?", ".", "", "NA", "~", None}`.
-
-    .. attribute:: compute_value
-
+        formats. Default is Variable.MISSING_VALUES.
+    compute_value : Callable
         A function for computing the variable's value when converting from
         another domain which does not contain this variable. The base class
         defines a static method `compute_value`, which returns `Unknown`.
         Non-primitive variables must redefine it to return `None`.
-
-    .. attribute:: source_variable
-
+    source_variable : Variable
         An optional descriptor of the source variable - if any - from which
         this variable is derived and computed via :obj:`compute_value`.
-
-    .. attribute:: attributes
-
+    attributes : dict
         A dictionary with user-defined attributes of the variable
-
-    .. attribute:: master
-
+    master : Variable
         The variable that this variable is a copy of. If a copy is made from a
         copy, the copy has a reference to the original master. If the variable
         is not a copy, it is its own master.
     """
     Unknown = ValueUnknown
+    MISSING_VALUES = {np.nan, "?", "nan", ".", "", "NA", "~"}
+
+    def __new__(cls, name="", *args, **kwargs):
+        # compatibility with str
+        return super().__new__(cls, name)
```

Comment:
Default is `str(..., encoding=sys.getdefaultencoding())`.  Will this work with non-ascii characters on Windos?